layers/+lang/typescript/packages.el

```diff
@@ -2,21 +2,42 @@
 ;;
 ;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
 ;;
-;; Author: Chris Bowdon <c.bowdon@bath.edu>
-;; URL: https://github.com/syl20bnr/spacemacs
+;; Author: JAremko <w3techplaygound@gmail.com> 
+;; URL: https://github.com/JAremko/spacemacs-pr
 ;;
```

Comment:
You should add your name, not delete his...you should never remove attributions!